ncov-recombinant/0.6.1/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM mambaorg/micromamba:0.24.0 as app
+
+USER root
+
+WORKDIR /
+
+#metadata
+LABEL base.image="FROM mambaorg/micromamba:0.24.0"
+LABEL dockerfile.version="1"
+LABEL software="ncov-recombinant"
+LABEL software.version="v0.6.1"
+LABEL description="SARS-CoV-2 recombinant sequence detection"
+LABEL website="https://github.com/ktmeaton/ncov-recombinant"
+LABEL license="https://github.com/ktmeaton/ncov-recombinant/blob/master/LICENSE"
+LABEL maintainer="Wes Hottel"
+LABEL maintainer.email="wesley-hottel@uiowa.edu"
+
+#Install dependencies
+RUN apt-get update && apt-get install -y python \ 
+  git \
+  wget && \
+  rm -rf /var/lib/apt/lists/* && apt-get autoclean
+
+#Clone ncov-recombinant repository 
+RUN git clone https://github.com/ktmeaton/ncov-recombinant.git &&\
+cd ncov-recombinant
```

Comment:
Instead of using git clone, could you download the tarball for the version you are creating an image for?

```
# something like
wget https://github.com/ktmeaton/ncov-recombinant/archive/refs/tags/v0.6.1.tar.gz
```

git clone is hard for version control

You image will also likely be smaller and more stable if you download the tarball and install it from that as well (instead of using micromamba). Did you already try that?